content/post/deux-sprints-une-version.en.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Two sprints, one version"
+publishdate: 2020-09-28
+draft: false
+tags: ["rero-ils", "release", "tests", "pilot libraries", "authorities"]
+slug: two-sprints-one-version
+---
+
+These last two months the RERO ILS team has worked successively on sprints [33](https://tree.taiga.io/project/rero21-reroils/taskboard/sprint-33-108) and [34](https://tree.taiga.io/project/rero21-reroils/taskboard/sprint-34-93) of the development. This work has resulted in a version `v0.12.0` offering many new features and deployed in time for the second phase of testing of the pilot libraries, which started at the end of September. Here are the major changes in this release, with full notes available on Github [`v0.12.0`](https://github.com/rero/rero-ils/blob/dev/RELEASE-NOTES.rst#v0120).
```

Comment:
Suggestion: remplacer
~~for the second phase of testing of the pilot libraries~~
par
for the second testing phase by the pilot libraries